c_cpp/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apt-get update && apt-get install -y --force-yes \
   && git clone https://github.com/google/googletest.git $HOME/googletest \
   && apt-get remove -y git \
   && apt-get autoremove -y \
+  && apt-get install -y wget \
```

Comment:
`wget` depends on `update` though, can i leave it in this `RUN` command and remove it in the same `RUN` command as the boost installation? Or to do `update` again and then deleting the files again during the boost installation?